_spinnaker/artifact_promotion.md

```diff
@@ -0,0 +1,116 @@
+
+You should now configure the expected artifacts for your pipeline and specify
+how your pipeline should be triggered. In our example we have specified a
+manifest (stored in our Github repo) as our artifact and set our pipeline to 
+trigger automatically when changes to our manifest are detected.
+
+![Artifact and Trigger](https://cl.ly/2807a66af53c/Image%2525202018-10-23%252520at%2525202.18.14%252520PM.png)
+
+Next, click `Add stage`. Provide your new stage a descriptive name -- we have
+chosen `Deploy to Staging` as ours. In order to deply our artifact, we choose 
+`Deploy (Manifest)` as our *Stage Type* and specify `Artifact` as our *Manifest
+Source*. Finally, we select our arfifact from the `Expected Artifact` dropdown.
+
+![Deploy (Manifest)](https://cl.ly/436328d30322/Image%2525202018-10-23%252520at%2525202.26.14%252520PM.png)
+
+Save your changes.
+
+### Validate your Deployment
+
+Now that we have a pipeline deploying to staging, we want validate that the 
+deployment was successful before promoting our artifact to production. 
+
+Click `Pipelines` and then click `+ Create` to create a new pipeline. We have
+named ours `Staging Judgement`.
+
+Add an automated trigger to the pipeline where the type is `Pipeline`. Select
+the `Deploy to Staging` pipeline from the `Pipeline` dropdown. Check both
+`successful` and `Trigger Enabled` from the remaining options.
+
+![Automated Triggers](https://cl.ly/cb93f2382dc2/Image%2525202018-10-23%252520at%2525202.30.55%252520PM.png)
+
+Now click, `Add stage` and select `Manual Judgement` as the *Stage Type*. In
+*Execution Options* select `halt the entire pipeline` for *If stage fails*. 
+Additionally, you can provide additional instructions in the *Instructions*
+text-area. 
+
+![Manual Judgement](https://cl.ly/a62c93f821d8/Image%202018-10-23%20at%201.47.21%20PM.png)
+
+Save your changes.
+
+### Deploy to Production
+
+We are now ready to create the pipeline responsible for deploying 
+our artifact to *production*. 
+
+Click `Pipelines` and then click `+ Create` to create a new pipeline. We have
+named this one `Deploy to Production` and have made use of the `Copy From` 
+dropdown in order to copy our `Deploy to Staging` pipeline.
+
+![Copy Staging Pipeline](https://cl.ly/d54d11e0c86f/Image%202018-10-23%20at%201.48.19%20PM.png)
+
+You will want to delete the *Automated Trigger* copied from our `Deploy to 
+Staging` pipeline. In its place, create a new one of *Type* `Pipeline`. In 
+the *Pipeline* dropdown select `Staging Judgement`. Lastly, select
+`successful` for *Pipeline Status* and select `Trigger Enabled`
+
+![Automated Trigger](https://cl.ly/321bce55b394/Image%202018-10-23%20at%201.50.56%20PM.png)
+
+Finally, add a new stage of type `Deploy (Manifest)`. The configuration for this 
+stage should mirror the configuration of the `Deploy (Manifest)` stage of
+the *Deploy to Staging* pipeline with the exception that specified account
+should correspond to your production environment. 
+
```

Comment:
Feels like we are missing an overview of how the three stages will now work together?  The bulk of the info has just been about creating the three pipelines, but not how it's intended to work (ie. why bother creating the manual judgment pipeline?)